boards/samr30-xpro/include/periph_conf.h

```diff
@@ -107,6 +111,10 @@ static const spi_conf_t spi_config[] = {
         .miso_pad = SPI_PAD_MISO_0,
         .mosi_pad = SPI_PAD_MOSI_2_SCK_3,
         .gclk_src = SAM0_GCLK_MAIN,
+#ifdef MODULE_PERIPH_DMA
+        .tx_trigger = SERCOM5_DMAC_ID_TX,
+        .rx_trigger = SERCOM5_DMAC_ID_RX,
+#endif
```

Comment:
That [vendor file](https://github.com/RIOT-OS/RIOT/blob/master/cpu/sam0_common/include/vendor/samr30/include/instance/sercom5.h#L136) is odd.
No DMA for SERCOM5?

Reply:
> That ~~[vendor file](https://github.com/RIOT-OS/RIOT/blob/master/cpu/sam0_common/include/vendor/samr30/include/instance/sercom5.h#L136)~~ **microcontroller** is odd.

![Screenshot_20200614_161725](https://user-images.githubusercontent.com/5160052/84595858-93e88400-ae5a-11ea-999c-d8c6d594794b.png)

> No DMA for SERCOM5?

No DMA for SERCOM5 :cry:

Reply:
Amended the commit to disable DMA for `SERCOM5` on the samr30-xpro